providers/jackson2/src/main/java/org/jboss/resteasy/plugins/providers/jackson/ResteasyJackson2Provider.java

```diff
@@ -72,16 +72,15 @@ public boolean isWriteable(Class<?> aClass, Type type, Annotation[] annotations,
 
    private static class ClassAnnotationKey
    {
-      private AnnotationArrayKey annotations;
+      private  Annotation[] annotations;
       private ClassKey classKey;
       private int hash;
```

Comment:
If we're making a change we should make these all final :)

Reply:
+1.  Fixed.